<?xml version="1.0"?>

<launch>
    <!-- Robot namespace -->
    <arg name="robot_name" default="my_gen3"/>
    <arg name="pose_input" default="/pcl_closest_obj/output_pose"/>
    <arg name="start_delay_seconds" default="0"/>

    <node pkg="robothon2023" type="slider_test.py" name="slider_test" output="screen">
        <rosparam command="load" file="$(find robothon2023)/config/joint_angles.yaml"/>
</node>
</launch>
